### PR TITLE
Add comments to fix sed compatibility issue between BSD and GNU versions

### LINKED ISSUE #645: Sed command in Deploy/jsminify.sh are not compatible between BSD and GNU versions
I think this is because sed BSD version is minor different between GNU version, and i will commit some comments to fix this compatibility issue.

[Answers on StackOverflow](http://stackoverflow.com/a/27834828/3167471)

### PR DESCRIPTION
**Pull request by ScorpioCPH**
fix #645

### CODE REVIEW

File: Deploy/jsminify.sh [hunk 1]
```diff
@@ -24,6 +24,13 @@ done
 echo "Remove space.."
 sed 's/ \+/ /g' -i /tmp/harbor.app.temp.js
 
+# If you want run this script on Mac OS X,
+# I suggest you install gun-sed (whth --with-default-names option).
```
Comment (reasonerjt) [inline Deploy/jsminify.sh]: is it gun-sed or gnu-sed?

File: Deploy/jsminify.sh [hunk 2]
```diff
@@ -24,6 +24,13 @@ done
...
+# $ brew install gnu-sed --with-default-names
+# Reference:
+# http://stackoverflow.com/a/27834828/3167471
+
+
```
Comment (reasonerjt) [inline Deploy/jsminify.sh]: remove this line please

File: Deploy/jsminify.sh (hunk 1 above)
Comment (ScorpioCPH) [inline Deploy/jsminify.sh]: oh, it is gnu-sed, sorry.

File: Deploy/jsminify.sh (hunk 2 above)
Comment (ScorpioCPH) [inline Deploy/jsminify.sh]: OK

**Comment by ScorpioCPH** [comment]
please use [2074af5](https://github.com/vmware/harbor/pull/647/commits/2074af56b377c47444351827b6c85df5a7c611c0)

**Comment by reasonerjt** [comment]
Thanks

**Pull request merged**